tests/incr_restore.py

```diff
@@ -80,9 +80,59 @@ def test_basic_incr_restore(self):
         # Clean after yourself
         self.del_test_dir(module_name, fname)
 
+    # @unittest.skip("skip")
+    def test_basic_incr_restore_1(self):
```

Comment:
I'd suggest using self-explanatory test names, instead of number suffixes.

Reply:
Done